sellercenter-category/src/main/java/com/gfg/sellercenter/category/entity/CategoryFulfillmentSetting.java

```diff
@@ -0,0 +1,21 @@
+package com.gfg.sellercenter.category.entity;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+@AllArgsConstructor
+@Getter
+@NoArgsConstructor
+public class CategoryFulfillmentSetting {
+    @NotNull
+    private int sellerId;
+
+    @NotNull
+    private int categoryId;
+
+    private Boolean fulfillmentEnabled;
```

Comment:
this should not be null, but it is nullable in db. Everything not explicitly truthy will be considered as false therefore.

Reply:
null means it will inherit from parent?

Reply:
no, "null" means someone broke the database. There should not be null, but column is nullable in DB definition. However I don't want consignment to stop work for seller just because of that. Therefore there are two options: stop generating templates at all for seller, or second: consider null values as falsy. I chose the second one.

Reply:
But when the category is broken the application should fail in my point of view. This is btw. the same behavior for DDD in promotion if you have incorrect data the whole thingy is broken.

Reply:
No problem, made it not nullable